crates/core/src/syn/parser/stmt/mod.rs

```diff
@@ -297,24 +297,28 @@ impl Parser<'_> {
 	///
 	/// # Parser State
 	/// Expects `USE` to already be consumed.
-	fn parse_use_stmt(&mut self) -> ParseResult<UseStatement> {
+	async fn parse_use_stmt(&mut self, stk: &mut Stk) -> ParseResult<UseStatement> {
 		let peek = self.peek();
 		let stmt = match peek.kind {
 			t!("NAMESPACE") => {
 				self.pop_peek();
-				let ns = self.parse_ident()?;
+				let ns = stk.run(|stk| self.parse_expr_field(stk)).await?;
 				if self.eat(t!("DATABASE")) {
-					let db = self.parse_ident()?;
+					let db = stk.run(|stk| self.parse_expr_field(stk)).await?;
 					UseStatement::NsDb(ns, db)
 				} else {
 					UseStatement::Ns(ns)
 				}
 			}
 			t!("DATABASE") => {
 				self.pop_peek();
-				let db = self.parse_ident()?;
+				let db = stk.run(|stk| self.parse_expr_field(stk)).await?;
 				UseStatement::Db(db)
 			}
+			t!("DEFAULTS") => {
```

Comment:
`USE DEFAULTS` does not read correct, maybe instead make it `USE DEFAULT` (no `S`)?